Merged with upstream PUP 4.9.0
(PUP-6795) Solaris does not support gzipped man pages
https://tickets.puppetlabs.com/browse/PUP-6795

--- a/install.rb
+++ b/install.rb
@@ -108,9 +108,12 @@ def do_man(man, strip = 'man/')
     FileUtils.makedirs(om, {:mode => 0755, :verbose => true})
     FileUtils.chmod(0755, om)
     FileUtils.install(mf, omf, {:mode => 0644, :preserve => true, :verbose => true})
-    gzip = %x{which gzip}
-    gzip.chomp!
-    %x{#{gzip} -f #{omf}}
+    # Solaris does not support gzipped man pages
+    unless Facter["os"].value["family"] == "Solaris"
+      gzip = %x{which gzip}
+      gzip.chomp!
+      %x{#{gzip} -f #{omf}}
+    end
   end
 end
